### ISSUE TITLE
guitar sniffer won't open

### ISSUE THREAD

**Issue by josh933** [artman41/guitarsniffer#9]
I've installed vjoy, usbpcap and winpcap, configured vjoy and tried to open guitar sniffer but it refuses to open. I've tried reinstalling it, running as admin, restarting my pc, adding as exception in my virus protection, and i've tried older versions but nothing seems to work.

**Comment by Dunkalunk** [comment]
Weird... The only things I've seen it keep from opening is if vJoy isn't available to be used (something else is using it or it's not installed), or if you don't have you Xbox One wireless adapter plugged in.
If the adapter isn't plugged in or the sniffer can't detect it then it won't open at all.
Other than that I'm not sure... I've never tried running it by command prompt but you could try that and see if it shows any other problems and post it here

**Comment by josh933** [comment]
I've been told that the guitar has to be flashed first with the hmxflashtool before guitar sniffer will open, but I can't even get that to work properly. the hmxtool opens, but I click ready and hold the red blue fret and xbox guide button but it won't do the update, it just turns the guitar off

**Comment by Dunkalunk** [comment]
Yeah I think I happened to have mine updated but didn't think that was required by any means.
If you're not sure how, look up how to run an exe file in a command prompt and let me know if it says any specific errors. If your wireless adapter is plugged in and the Guitar is connected then that's the next step I think

**Comment by ArlanPrado** [comment]
I solved this issue by temporarily setting my program folders to the C drive as I had it on a different drive. Make sure to restart your computer after setting the program folders just in case and also restart after installing vjoy. Just make sure the 3 programs are in your C drive program folders.
